cpp/src/aztec/plonk/composer/standard_composer.test.cpp

```diff
@@ -13,65 +13,65 @@ auto& engine = numeric::random::get_debug_engine();
 
 TEST(standard_composer, base_case)
```

Comment:
This file seems like a good candidate for a `using namespace plonk` or otherwise putting these tests themselves in the `plonk` namespace. Then we don't have to qualify all of the names--it really does seem redundant here since we're testing stuff in the `plonk` namespace.

Reply:
Similarly for other test files that follow. But maybe I'm overlooking something?

Reply:
A great observation. Done.